apps/mudita-center-e2e/wdio.conf.ts

```diff
@@ -78,12 +79,13 @@ export const config: Options.Testrunner = {
       toRelativePath(TestFilesPaths.backupLocationTest),
       toRelativePath(TestFilesPaths.mcCheckForUpdatesOfflineTest),
       toRelativePath(TestFilesPaths.homePageTestDeviceNotConnectedTest),
-      toRelativePath(TestFilesPaths.newsPageOnlineTest),
```

Comment:
You deleted online test from someone else

Reply:
problem is not occuring on current commit / fixed